packages/webcomponents/src/components/payouts-list/get-payouts.ts

```diff
@@ -3,10 +3,10 @@ import { ComponentErrorSeverity } from '../../api/ComponentError';
 import { getErrorCode, getErrorMessage } from '../../api/services/utils';
 
 export const makeGetPayouts =
-  ({ id, authToken, service }) =>
+  ({ id, authToken, service, apiOrigin }) =>
```

Comment:
I started to get a TS error on `src/components/payouts-list/test/get-payouts.spec.ts:32`

We probably should either pass the `apiOrigin` in the test file, or add a default value in here.